Project/Assets/ML-Agents/Examples/3DBall/Scripts/Ball3DAgent.cs

```diff
@@ -9,6 +9,7 @@ public class Ball3DAgent : Agent
 {
     [Header("Specific to Ball3D")]
     public GameObject ball;
+    public bool useVecObs;
```

Comment:
I would add a Tooltip to explain that this needs to be checked in the 3DBall scene but unchecked in the Visual3DBall scene (since it is a public inspector).